lib/modern_searchlogic/column_conditions.rb

```diff
@@ -102,6 +102,26 @@ def searchlogic_association_suffix_match(method_name)
         end
       end
 
+      def searchlogic_column_boolean_match(method_name)
+        if match = method_name.match(/^not_(.*)/)
+          column_name = match[1]
+          if boolean_column?(column_name)
+            return {:arity => 0, :block => lambda do
+              where(column_name => false)
+            end}
+          end
+        elsif boolean_column?(method_name)
+          return {:arity => 0, :block => lambda do
```

Comment:
i realize you're just mimicking the style from below (which I'm now 🤦‍♂ing about putting in there) but can you remove the `return` keywords?

Reply:
@a-warner, a separate has been put up for that at PR #6

Reply:
great, approved this one!